src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/PluginConfigTest.java

```diff
@@ -38,33 +43,74 @@
     //CS IGNORE MagicNumber FOR NEXT 100 LINES. REASON: Mocks tests.
 
     /**
-     * test.
+     * Test creation of a config object from form data.
+     * filterIn using partial event list.
      */
     @Test
     public void testSetValues() {
+        String events = "change-abandoned change-merged change-restored";
         String formString = "{"
                 + "\"numberOfSendingWorkerThreads\":\"4\","
-                + "\"numberOfReceivingWorkerThreads\":\"6\"}";
+                + "\"numberOfReceivingWorkerThreads\":\"6\","
+                + "\"filterIn\":\"" + events + "\"}";
```

Comment:
Since you are modifying static data in this test (as the tests in gerrit-events) We probably need to make sure they are reset to default between tests to cover our behinds in the future.
I think it can be done with an `@After` method in this class.

Reply:
Done